src/components/TopBar/README.md

```diff
@@ -256,7 +254,6 @@ function TopBarExample() {
   return (
     <div style={{height: '250px'}}>
       <AppProvider
-        theme={theme}
```

Comment:
Now this has been removed the AppProvider is the same as the one that all readmes examples get by default. I think this means we can get rid of `omitAppProvider: true` and get rid of all the <AppProvider> wrappers in the examples